evennia/commands/cmdsethandler.py

```diff
@@ -351,6 +351,14 @@ def update(self, init_mode=False):
                     elif path:
                         cmdset = self._import_cmdset(path)
                         if cmdset:
+                            if cmdset.key == '_CMDSET_ERROR':
+                                # If a cmdset fails to load, check if we have a fallback path to use
+                                fallback_path = settings.CMDSET_FALLBACKS.get(path, None)
```

Comment:
I like the look of this, in the sense that we won't wreck the reload process on a faulty cmdset string. I think we are swallowing the error here though; if we are falling back to a default cmdset the user may think all works fine yet they will get the fallback version of commands without realizing it. See for example the problems people have with us falling back to the `DefaultObject` when a typeclass fails to load - even though we log it. I think that at a minimum we need to use `logger.log_errmsg()` to forward which cmdset failed to load here.